spec/controllers/v1/queue_items_controller_spec.rb

```diff
@@ -16,7 +16,35 @@
       expect(policy).to eq QueueItemPolicy
     end
 
-    it_behaves_like "an index endpoint", 'QueueItemsPolicy'
+    describe "GET #index" do
```

Comment:
Three of these tests are just extracted from the "an index endpoint" shared examples, one of which is modified. I'm fine with that for now. However, it may make sense to not perform setup within the shared examples, instead relying on the calling scope to properly setup the context.